mlir/test/Dialect/Vector/vector-unroll-options.mlir

```diff
@@ -497,6 +497,45 @@ func.func @elementwise_4D_to_2D(%v1: vector<2x2x2x2xf32>, %v2: vector<2x2x2x2xf3
 // CHECK-NOT: arith.addf
 // CHECK: return
 
+func.func @vector_create_mask(%size1: index, %size2: index) -> vector<16x16xi1> {
+  %0 = vector.create_mask %size1, %size2 : vector<16x16xi1>
```

Comment:
please, add a test with (full or partial?) constant indices to make sure that folding happens